### PR TITLE
Jumbotron refinements and countdown refactor/redesign (HEHA-587)

### PR DESCRIPTION
**Pull request by nassimdonald**
#### What does this PR do? (please provide any background)

**Countdown**
* Redesigned countdown to simple text-based format – font-sizes and styles to be defined using classes (e.g. `<span class="countdown-value h1">30</span>`) and/or in app-specific LESS, if necessary – see corresponding PR in Pattern Library.

**Jumbotron**
* Removed excessive responsive font-sizes in Jumbotron – having too many generic font-size overrides feels hacky.
* Applied text-shadow to Jumbotron based on luminosity of text.
* Defined better margins for headings in Jumbotrons – temporary measure until we can address typography and spacing at a global level.
* Increased jumbotron padding.
* Flipped and modified jumbotron overlay gradient.

**Misc**
* Refactored weather `<span>` to use `.sr-only` mixin for screenreader accessibility.

#### What tests does this PR have?
None.

#### How can this be tested?
1. Checkout the branch `heha-587` in both the Brands repo and the Pattern Library repo.
2. In the directory of Pattern Library, run `npm link {path-to-your-brands-repo}`.
3. In Pattern Library, run `npm run build` followed by `npm run start`.
4. Visit the [Planner](http://localhost:3000/pages/planner/index.html) page in Pattern Library.

#### What domains are affected?
Any that currently use the weather component and/or the HX-branded Jumbotron.

#### Any tech debt?
Not really tech debt, but we need to choose better images for the Trip Planner – less busy and lower contrast.

#### Screenshots / Screencast
trip planner - small jumbotron - desktop v2
trip planner - small jumbotron - mobile v2

#### What gif best describes how you feel about this work?

- [x] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Notes**

When reviewing Static Site Generator projects, please think carefully about what can be done to centralise common functionality. Please also pay attention to anything configurable that will also need updating in the Yeoman generator template

---

**Reviewer**
- [x] :+1:
- [ ] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

---

### CODE REVIEW

File: brands/globals/src/less/toolkit/components/_weather.less [hunk 1]
```diff
@@ -46,10 +46,6 @@
     visibility: visible;
   }
   span {
-    display: block;
-    text-indent: 100%;
-    white-space: nowrap;
-    overflow: hidden;
-    height: 0;
+    .sr-only;
```
Comment (stuartgiles) [inline brands/globals/src/less/toolkit/components/_weather.less]: This class could be applied in the html to save generating additional css

Comment (nassimdonald) [inline brands/globals/src/less/toolkit/components/_weather.less]: I'm undecided as to whether it is better to keep the markup of new components simple and define everything in CSS (e.g. columns, font sizes, colours etc.), or vice-versa…

Helper classes such as `sr-only` probably make sense to belong in markup, for better clarity of what elements are expected to be invisible. But perhaps things like padding, margins, font-size etc. belong in the CSS – even if they can be achieved with classes.

@jodiedoubleday: What are your thoughts on the matter?

**Comment by h0rhay** [comment]
I've seen this working locally, looks good on desktop and devices with chrome devtools.
Happy to 👍 when @jodiedoubleday feeds back on the .sr-only class

File: brands/globals/src/less/toolkit/components/_weather.less (hunk 1 above)
Comment (jodiedoubleday) [inline brands/globals/src/less/toolkit/components/_weather.less]: I think its probably best to put this in the markup. It give clarity to why something is or isn't appearing. and lets people know its meant o be hidden.

**Comment by nassimdonald** [comment]
@h0rhay: Ready to go. Thanks!

File: brands/globals/src/less/toolkit/components/_weather.less (hunk 1 above)
Comment (nassimdonald) [inline brands/globals/src/less/toolkit/components/_weather.less]: Just to note, with the current limitations of using ui-toolkit, we can't freely add classes to components. Therefore we have had to add the `sr-only` styling in the LESS.

**Pull request merged**